docs/modules/plugins/pages/plugin-applitools.adoc

```diff
@@ -1 +1,223 @@
+
+image::applitools_establish_view.png[Establish view]
+
+Established baseline details
+
+image::applitools_establish_step_editor.png[Establish step editor]
+
+include::partial$compare-against.adoc[]
+
+.Compare against established baseline
+[source,gherkin]
+----
+Given I am on a page with the URL 'https://example.com/'
+When I run visual test with Applitools using:
+|appName    |batchName      |baselineName |action         |elementsToIgnore|
+|example.com|example-testing|main-app-page|COMPARE_AGAINST|By.tagName(h1)  |
+----
+
+Comparison result against the established baseline on Applitools
+
+image::applitools_compare_view.png[Compare view]
+
+Comparison result details
+
+image::applitools_compare_step_editor.png[Compare step editor]
+
+include::partial$check-inequality-against.adoc[]
+
+.Check inequality established baseline
+[source,gherkin]
+----
+Given I am on a page with the URL 'https://example.com/'
+When I run visual test with Applitools using:
+|appName    |batchName      |baselineName |action                  |elementsToIgnore|
+|example.com|example-testing|main-app-page|CHECK_INEQUALITY_AGAINST|By.tagName(h1)  |
+----
+
+== Steps
+
+=== Visual check with default applitools configuration
+
+Performs visual check on the Applitools visual testing plarform.
+
+[source,gherkin]
+----
+When I $actionType baseline `$testName` in batch `$batchName` with Applitools
+----
+
+* `$actionType` - The {visual-testing-actions}.
+* `$testName` - The name of the baseline (visual test name).
+* `$batchName` - The name of the batch (group of visual tests).
+
+.Perform visual check using default configuration
+[source,gherkin]
+----
+When I ESTABLISH baseline `main-app-page` in batch `example-testing` with Applitools
+----
+
+=== Visual check with custom applitools configuration
+
+Performs visual check on the Applitools visual testing plarform with a custom applitools configuration.
+
+Properties in the custom applitools configuration provided in step take presendence over properties specified in configuration files.
+
+[source,gherkin]
+----
+When I run visual test with Applitools using:$applitoolsConfigurations
+----
+
+* `$applitoolsConfigurations` - The Applitools configuration
+** `action` - The {visual-testing-actions}.
+** `batchName` - The name of the batch (group of visual tests).
+** `baselineName` - The name of the baseline (visual test name).
+** `matchLevel` - The https://applitools.com/docs/common/cmn-eyes-match-levels.html#Available[match level].
+** `appName` - The name of the application under test.
+** `elementsToIgnore` - The comma separated list of locators of elements to ignorer.
+** `areasToIgnore` - The comma separated list of locators of page areas to ignorer.
+** `hostApp` - The application running the AUT (e.g. Safari, Chrome).
+** `hostOS` - The host OS running the AUT.
+** `viewportSize` - The required viewport size for the environment.
+** `baselineEnvName` - The name of the baseline's environment.
```

Comment:
it's potentially dangerous property to specify :)

Reply:
agree :) but it's exposed to end users so we have to document this :)